### PR TITLE
#37 - change social medias icons design

### LINKED ISSUE #37: Change social medias icons design
Closed with PR https://github.com/blumilksoftware/laravel-poland-meetup/pull/38

Change social medias icons design. Unify design and animations

### PR DESCRIPTION
**Pull request by paulinaszkudlarek**
this should be closed after #39, this should close #37 
Unify colors, add slideshare icon, add hover effect

image

### CODE REVIEW

File: frontend/components/Companies/Counters.vue [hunk 1]
```diff
@@ -12,55 +14,49 @@ const props = defineProps({
...
+  if (props.meetups.length === 0) return {}
+  const { lists } = useBuildLists(props.meetups, props.company)
+  console.log('lists.value', lists.value)
+  return lists.value
 })
 
+
 </script>
 <template>
+
```
Comment (EwelinaSkrzypacz) [inline frontend/components/Companies/Counters.vue]: Please read warning from linter and fix all issues.

File: frontend/components/Companies/Counters.vue [hunk 2]
```diff
@@ -12,55 +14,49 @@ const props = defineProps({
...
       <p class="ml-2 self-center">
         prezentacje
       </p>
-    </div>
-    <div v-if="speakers.size > 0">
+      <!-- </div> -->
+    </router-link>
+    <router-link v-if="buildLists.speakers"
+      :to="{ name: 'companies.details.list', params: { id: company, tab: 'speakers' } }">
+      <!-- <div v-if="buildLists.speakers"> -->
```
Comment (EwelinaSkrzypacz) [inline frontend/components/Companies/Counters.vue]: Do we really need this comment?

File: frontend/pages/Companies.vue [hunk 3]
```diff
@@ -23,5 +23,5 @@ onMounted(async () => {
   <page-header word1="Organizatorzy" word2="i" word3="partnerzy" sentence="Dołącz do nas i promój swoją firmę!"/>
```
Comment (EwelinaSkrzypacz) [inline frontend/pages/Companies.vue]: It should be "promuj".

File: frontend/components/Icons/LinkedinIcon.vue [hunk 4]
```diff
@@ -9,7 +9,8 @@ defineProps({
 
 <template>
   <a :href="href" class="transition duration-200 hover:scale-110 hover:text-[#0A66C2]">
-    <svg id="Icons" fill="currentColor" height="35px" width="35px" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 32 32" xml:space="preserve">
+    <span class="sr-only">Linkedin</span>
```
Comment (EwelinaSkrzypacz) [inline frontend/components/Icons/LinkedinIcon.vue]:
```suggestion
    <span class="sr-only">LinkedIn</span>
```

File: frontend/components/Icons/LinkedinIcon.vue [hunk 5]
```diff
@@ -9,7 +9,8 @@ defineProps({
...
+    <svg id="Icons" fill="currentColor" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 32 32" xml:space="preserve">
```
Comment (EwelinaSkrzypacz) [inline frontend/components/Icons/LinkedinIcon.vue:13]: I think id should be more descriptive, something like "LinkedIn".

**Review by EwelinaSkrzypacz — COMMENTED** [review]
Here it's bad linking:

.../meetups/2022-11-10-laravel-poland-meetup-23

image

File: frontend/components/Companies/Counters.vue (hunk 2 above)
Comment (paulinaszkudlarek) [reply frontend/components/Companies/Counters.vue]: nope and it is removed in #39

File: frontend/pages/Companies.vue (hunk 3 above)
Comment (paulinaszkudlarek) [reply frontend/pages/Companies.vue]: it is changed in #39 :P

File: frontend/components/Icons/LinkedinIcon.vue (hunk 5 above)
Comment (paulinaszkudlarek) [reply frontend/components/Icons/LinkedinIcon.vue:13]: right, I just commited this change.

**Comment by paulinaszkudlarek** [comment]
@EwelinaLasowy i checked all conversations and all of them can be resolved. Eslint did its job as well. You can do the review

File: frontend/components/Icons/SlideShareIcon.vue [hunk 6]
```diff
@@ -1,8 +1,20 @@
+<script setup>
+defineProps({
+  href: {
+    type: String,
+    default: '',
+  },
+})
+
```
Comment (MichalMyskow) [inline frontend/components/Icons/SlideShareIcon.vue]:
```suggestion
```

**Review by MichalMyskow — APPROVED** [review]

**Review by EwelinaSkrzypacz — APPROVED** [review]

**Pull request merged**